R/read.quitte.R

```diff
@@ -57,6 +58,9 @@ read.quitte <- function(file,
     .read.quitte <- function(f, sep, quote, na.strings, convert.periods,
                              drop.na, comment) {
 
+        if (grepl("\\.xlsx$", f)) {
+            return (as.quitte(read_excel(path = f, sheet = "data")))
+        }
```

Comment:
Why so restrictive?  Could use `'\\.xlsx?$'` to match both kinds of Excel files and `sheet = ifelse('data' %in% exel_sheets(f), 'data', 1)` to read either the data sheet if present or the first sheet, giving users some leeway.